error/s2n_errno.c

```diff
@@ -317,7 +322,7 @@ const char *s2n_strerror(int error, const char *lang)
         case S2N_ERR_T_USAGE_END:
             break;
 
-        /* No default to make compiler fail on missing values */
+            /* No default to make compiler fail on missing values */
```

Comment:
This seems wrong.

Reply:
Yeah. I suspect this is a result of how clang-format works. -- It doesn't understand that the break; ends the scope for the switch statement. -- Looking for a solution.

Reply:
I couldn't find anything satisfactory.

I vote we live with it.

The alternative is really bad:

```c
    break;
        /* clang-format off */
     /*  No default to make compiler fail on missing values */
       /* clang-format on */
```

In-lining the clang-format directives doesn't work. The clang-format comments always get formatted themselves so they will end up with the weird formatting.

Reply:
Yeah we can live with it. I don't think it actually impacts readability.